### PR TITLE
Prepare for adding service-proxy-name label for half of the services

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/perf-tests/issues/413

### CODE REVIEW

**Comment by wojtek-t** [comment]
/hold

This requires some testing.

File: clusterloader2/testing/load/service.yaml [hunk 1]
```diff
@@ -2,6 +2,10 @@ apiVersion: v1
 kind: Service
 metadata:
   name: {{.Name}}
+{{if IsEven 1}}
```
Comment (krzysied) [inline clusterloader2/testing/load/service.yaml]: `{{if IsEven .Index}}`?

File: clusterloader2/pkg/config/template_functions.go [hunk 2]
```diff
@@ -168,6 +170,14 @@ func minFloat(numbers ...interface{}) float64 {
 	return min
 }
 
+func isEven(number interface{}) bool {
+	return int(toFloat64(number)) % 2 == 0
+}
+
+func isOdd(a interface{}) bool {
```
Comment (krzysied) [inline clusterloader2/pkg/config/template_functions.go]: `func isOdd(number interface{}) bool {`

Comment (wojtek-t) [reply clusterloader2/pkg/config/template_functions.go]: done

File: clusterloader2/testing/load/service.yaml (hunk 1 above)
Comment (wojtek-t) [reply clusterloader2/testing/load/service.yaml]: Long-term yes. But if I would do it now, I would basically remove half of existing services from kube-proxy - I don't want to do this now.

Comment (krzysied) [reply clusterloader2/testing/load/service.yaml]: Ok.

**Comment by wojtek-t** [comment]
Verified that it's no op now and will work when changed to .Index

/hold cancel

@krzysied - PTAL since I fixed gofmt.

**Pull request merged**